packages/web/src/fixtures/dev-for-apps/functions/useUploadFile.ts

```diff
@@ -51,22 +51,22 @@ export const useDeleteFile = (accountAddress: string) => {
   const [isLoading, setIsLoading] = useState<boolean>(false)
 
   const deleteFileHandler = async (id: number, filename: string) => {
-    const signMessage = `delete file: ${id}, ${filename}`
+    const signMessage = `Delete file: ${id}, ${filename}`
     const { signature, message: signedMessage } = await signWithCache(web3, signMessage)
     if (!signature || !signedMessage) {
       return
     }
 
     setIsLoading(true)
-    message.loading({ content: 'delete file...', duration: 0, key })
+    message.loading({ content: 'Deleting File...', duration: 0, key })
 
     const res = deleteUploadFile(signedMessage, signature, accountAddress, id)
       .then(result => {
```

Comment:
Unrelated, but any reasons you're not using async/await? Would make the code much more readable.

Reply:
Thanks for the PR!

By the way, there should be no particular reason not to use async/await here. We can refactor it. Thanks for pointing that out!

Reply:
I would recommend the https://github.com/xjamundx/eslint-plugin-promise/blob/development/docs/rules/prefer-await-to-then.md rule. It lets you automate the enforcement.